examples/lfric/scripts/async_halo_exchanges.py

```diff
@@ -32,6 +32,7 @@
 # POSSIBILITY OF SUCH DAMAGE.
 # -----------------------------------------------------------------------------
 # Authors: R. Ford, A. R. Porter and S. Siso, STFC Daresbury Laboratory
+# Modified by O. Brunt, Met Office
```

Comment:
I would suggest `# Modified: O. Brunt, Met Office` for consistency with the "Authors" line above.